### PR TITLE
EOS-28986: UI bug fixes during the internal discussion

### PR DESCRIPTION
**Pull request by NareshGopal**
# Problem Statement
- UI bugs were fixed on call during our internal discussion.

# Design
-  Minor bugs were fixed during the internal review done before the demo

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/App.constant.ts [hunk 1]
```diff
@@ -15,48 +15,49 @@
...
+      requiredAccess: "sysconfig",
+    },
+    {
+      title: "manage",
+      path: "/manage",
+      iconDefault: "images/manage-grey.svg",
+      iconActive: "images/manage-white.svg",
+      requiredAccess: "s3accounts",
+    },
+    //Uncomment the below menu items as it is built
```
Comment (sujit-h-singh) [inline src/App.constant.ts:40]: You can revert these changes as demo is completed.

Comment (NareshGopal) [reply src/App.constant.ts]: Done. I have also added the missing password show/hide buttons.

**Review by sujit-h-singh — APPROVED** [review]
